packages/Provider/src/Provider.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+
+import { I18nProvider, type I18nProviderProps, useLocale, useLocalizedStringFormatter } from "react-aria";
+
+import "./provider.scss";
```

Comment:
remove this line, there is no scss

Reply:
I originally had done this, but it kept complaining that there was no provider.css in the dist file... I removed all references to it as far as I know, it might have been because we use it in the decorator..

Reply:
and delete the .scss file as well. No need to keep stuff just in case

Reply:
I'll investigate, there is no reason this file should exist.